### ISSUE TITLE
Eliminate Ports S_AXI_C* Until Supported by AutoFPGA

### ISSUE THREAD

**Issue by bobnewgard** [ZipCPU/wb2axip#40]
In axidmacheck, Verilator v4.202 fails to compile main.v due to
unconnected axidma ports S_AXI_C*.  To fix this I eliminated
S_AXI_C* ports in wb2axip/rtl/axidma.v.  Details below.

Regards,
Bob Newgard

Details:
* https://github.com/ZipCPU/autofpga/commit/f89d4dce81baed312c843c8554923fee7750d0a1
* https://github.com/bobnewgard/axidmacheck/commit/af7511a857310634db00f5a0e10a19abd159be3e
* https://github.com/bobnewgard/wb2axip/commit/e918a8a5b192d9889a1fb2cfafe2bd507c69791a
* Verilator v4.202

**Comment by ZipCPU** [comment]
Yes, the AXIDMA S_AXI_C* ports need to be removed.  Your fix will work temporarily as a patch, but I'd like to maintain the ability to shut the clock down--just without the S_AXI_C* ports that never really added anything to that functionality.  So expect an update to come, just ... not quite this one.
